machine-learning/01-openpbta_heatmap.Rmd

```diff
@@ -0,0 +1,383 @@
+---
+title: "OpenPBTA: Create a heatmap"
+output:   
+  html_notebook: 
+    toc: true
+    toc_float: true
+author: CCDL for ALSF
+date: 2020
+---
+
+In this notebook, we cluster RNA-seq data from the Open Pediatric Brain Tumor Atlas (OpenPBTA) project and create a heatmap.
+OpenPBTA is a collaborative project organized by the CCDL and the Center for Data-Driven Discovery in Biomedicine (D3b) at the Children's Hospital of Philadelphia conducted openly on GitHub.
+
+You can read more about the project [here](https://github.com/alexslemonade/openpbta-analysis/#openpbta-analysis).
+
+We've downloaded the publicly available data from the project and performed a variance stabilizing transformation on the stranded RNA-seq dataset.
+(That code is available in `setup`!)
+
+We'll use a package called [`ComplexHeatmap`](https://www.bioconductor.org/packages/release/bioc/html/ComplexHeatmap.html) to make our heatmap.
+This package also will perform clustering as part of the heatmap.
+It is highly flexible; see the [`ComplexHeatmap` Complete Reference](https://jokergoo.github.io/ComplexHeatmap-reference/book/) for more information.
```

Comment:
Warn also that it is opinionated? You have to get the data structures *just right*